yarn.lock

```diff
@@ -12784,38 +12857,29 @@ sugarss@^2.0.0:
     postcss "^7.0.2"
 
 summernote@^0.8.2:
-  version "0.8.10"
-  resolved "https://registry.yarnpkg.com/summernote/-/summernote-0.8.10.tgz#21a5d7f18a3b07500b58b60d5907417a54897520"
+  version "0.8.12"
```

Comment:
@jayoshih Please see https://github.com/learningequality/studio/pull/1533

Reply:
@MisRob Thanks for pointing this out! I can merge your PR to the new vue-refactor branch and pull from that